.github/workflows/native-wsl.yml

```
@@ -0,0 +1,153 @@
+name: Native and WSL
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    defaults:
+        run:
+          shell: ${{ matrix.configuration == 'wsl' && 'wsl-bash {0}' || 'pwsh' }}
+    strategy:
+      matrix:
+        os: [windows-2019]
+        node-version: [18, 16, 14, 12, 10, 8, 6, 4]
+        configuration: [wsl, native]
+    steps:
+    - uses: actions/checkout@v4
+    - uses: Vampire/setup-wsl@v3
+      if: matrix.configuration == 'wsl'
+      with:
+        distribution: Ubuntu-22.04
+    - name: Do everything here because npm is not found when running in separate steps
+      if: matrix.configuration == 'wsl'
+      run: |
+        curl --version
+        export ESLINT_VERSION=7
+        export TRAVIS_NODE_VERSION=${{ matrix.node-version }}
+        curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/master/install.sh | bash
+        export NVM_DIR="$HOME/.nvm"
+        [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+        [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+        nvm install ${{ matrix.node-version }}
+        nvm use ${{ matrix.node-version }}
+        nvm install-latest-npm
+
+        if [ ${{ matrix.node-version }} -ge 4 -a ${{ matrix.node-version }} -lt 6 ]; then
+          npm install eslint@4 --no-save --ignore-scripts
+          npm install
+          npm install eslint-import-resolver-typescript@1.0.2 --no-save
+          npm uninstall @angular-eslint/template-parser --no-save
+          npm uninstall @typescript-eslint/parser --no-save 
+        fi
+        if [ ${{ matrix.node-version }} -ge 6 -a ${{ matrix.node-version }} -lt 7 ]; then
+          npm install eslint@5 --no-save --ignore-scripts
+          npm install
+          npm install eslint-import-resolver-typescript@1.0.2 --no-save
+          npm uninstall @angular-eslint/template-parser --no-save
+          npm install @typescript-eslint/parser@3 --no-save 
```

```suggestion
          npm uninstall @angular-eslint/template-parser --no-save
          npm install eslint-import-resolver-typescript@1.0.2 @typescript-eslint/parser@3 --no-save 
```
assuming this still works?